src/opera/pge/dswx_s1/dswx_s1_pge.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+
+"""
+==============
+dswx_s1_pge.py
+==============
+Module defining the implementation for the Co-registered Single Look Complex (DSWX)
```

Comment:
"Co-registered Single Look Complex" should be changed to "Dynamic surface water extent"